yorkie/src/main/kotlin/dev/yorkie/document/Document.kt

```diff
@@ -50,9 +53,16 @@ import kotlinx.coroutines.withContext
 /**
  * A CRDT-based data type.
  * We can represent the model of the application and edit it even while offline.
+ *
+ * A single-threaded, [Closeable] [dispatcher] is used as default.
+ * Therefore you need to [close] the client, when the client is no longer needed.
```

Comment:
it's not that crucial, but I think `client` should be updated to `document` in the comment.
```suggestion
 * Therefore you need to [close] the document, when the document is no longer needed.
```